cmd/describe.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+	"github.com/redhat-developer/ocdev/pkg/project"
+	"github.com/redhat-developer/ocdev/pkg/storage"
+	"github.com/redhat-developer/ocdev/pkg/url"
+	"github.com/spf13/cobra"
+)
+
+var componentDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe the given application",
```

Comment:
Describe component not application.

can you add nicer messages and examples?

Something like what we have for create
https://github.com/redhat-developer/ocdev/blob/master/cmd/create.go#L26